docs/types/builtin-function.md

```diff
@@ -212,6 +212,60 @@ Halts the process for as many `ms` you specified:
 sleep(1000) # sleeps for 1 second
 ```
 
+### source(fileName) -- aka src(), include(), import(), or require() 
+
+Evaluates the ABS script `fileName` in the context of the ABS global
+environment. The results of any expressions in the file become
+available to other commands in the REPL command line or to other
+scripts in the current script execution chain. 
```

Comment:
I want to expand a bit on why one would use the `source` function here -- just a heads up that I'll add more documentation later on.

Reply:
OK